tools/build-vagrant-box.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -ex
+
+WORKDIR=$(mktemp -d)
+
+function cleanup {
+    rm -rf "${WORKDIR}"
+}
+
+trap cleanup EXIT
+
+sudo apt-get update
+sudo apt-get install -y --no-install-recommends \
+    curl \
+    unzip
+
+git clone https://github.com/jakobadam/packer-qemu-templates.git ${WORKDIR}
+
+cd ${WORKDIR}/ubuntu
+
+packer build -var-file=ubuntu1604.json ubuntu-vagrant.json
```

Comment:
I agree this is a good idea.  Is that something you would be comfortable doing as a separate PR while I'm gone the next two weeks?